release-notes/opensearch.release-notes-3.0.0-alpha1.md

```diff
@@ -46,6 +46,7 @@
 - [Star Tree] [Search] Resolving keyword & numeric bucket aggregation with metric aggregation using star-tree ([#17165](https://github.com/opensearch-project/OpenSearch/pull/17165))
 - Added error handling support for the pull-based ingestion ([#17427](https://github.com/opensearch-project/OpenSearch/pull/17427))
 - Added Warm index setting and Updated nomenclature to differentiate between hot and warm tiering implementation ([#17490](https://github.com/opensearch-project/OpenSearch/pull/17490))
+<<<<<<< HEAD:release-notes/opensearch.release-notes-3.0.0-alpha1.md
```

Comment:
This looks like residue from a merge conflict.

Also, we shouldn't add this to the 3.0.0-alpha1 release notes anyway, since we're not including it in 3.0.0-alpha1. It should just go in `CHANGELOG.md`.

Reply:
resolved